### ISSUE TITLE
React: Masthead - Missing Label for search edit combo

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5190]
## Detailed description

On  JAWS FF/Voiceover iPhone

Issue:
Missing Label for search edit combo

Note: Label is present in Web component staging
